plugins/window-state/src/lib.rs

```diff
@@ -320,6 +321,7 @@ impl<R: Runtime> WindowExtInternal for Window<R> {
 #[derive(Default)]
 pub struct Builder {
     denylist: HashSet<String>,
+    filter_callback: Option<Arc<Mutex<FilterCallbackFn>>>,
```

Comment:
I think just `Option<Box<FilterCallbackFn>>` would be enough here?

Reply:
Yes. Fixed.